spmd/tensor/__init__.py

```diff
@@ -77,3 +77,13 @@ def distribute_tensor(
         placements,
         requires_grad=tensor.requires_grad,
     )
+
+
+# All public APIs from dtensor package
+__all__ = [
```

Comment:
Will adding them to multiple packages introduce confusion to users? For example, `spmd.DTensor` and `spmd.tensor.DTensor`.

Reply:
good point, I'll remove the spmd.DTensor import.